definitions/npm/cb-fetch_v1.11.x/flow_v0.104.x-/cb-fetch_v1.11.x.js

```diff
@@ -0,0 +1,121 @@
+declare module 'cb-fetch' {
+
+    declare module.exports: typeof request;
+
+    declare type RequestBody =
+      | BodyInit
```

Comment:
`BodyInit` doesn't seem to be declared in the `104.x` def file.

Reply:
It's declared [here](https://github.com/facebook/flow/blob/master/lib/bom.js#L1398).